arangod/VocBase/Methods/UpgradeTasks.cpp

```diff
@@ -307,50 +354,93 @@ bool UpgradeTasks::insertRedirections(TRI_vocbase_t* vocbase,
       THROW_ARANGO_EXCEPTION(opres.result);
     }
   }
+
   res = trx.finish(opres.result);
+
   if (!res.ok()) {
     THROW_ARANGO_EXCEPTION(res);
   }
+
   return true;
 }
 
-bool UpgradeTasks::setupAqlFunctions(TRI_vocbase_t* vocbase,
-                                     VPackSlice const&) {
-  return ::createSystemCollection(vocbase, "_aqlfunctions");
+bool UpgradeTasks::setupAqlFunctions(
+    TRI_vocbase_t& vocbase,
+    arangodb::velocypack::Slice const& slice
+) {
+  return ::createSystemCollection(&vocbase, "_aqlfunctions");
 }
 
-bool UpgradeTasks::createFrontend(TRI_vocbase_t* vocbase, VPackSlice const&) {
-  return ::createSystemCollection(vocbase, "_frontend");
+bool UpgradeTasks::createFrontend(
+    TRI_vocbase_t& vocbase,
+    arangodb::velocypack::Slice const& slice
+) {
+  return ::createSystemCollection(&vocbase, "_frontend");
 }
 
-bool UpgradeTasks::setupQueues(TRI_vocbase_t* vocbase, VPackSlice const&) {
-  return ::createSystemCollection(vocbase, "_queues");
+bool UpgradeTasks::setupQueues(
+    TRI_vocbase_t& vocbase,
+    arangodb::velocypack::Slice const& slice
+) {
+  return ::createSystemCollection(&vocbase, "_queues");
 }
 
-bool UpgradeTasks::setupJobs(TRI_vocbase_t* vocbase, VPackSlice const&) {
-  return ::createSystemCollection(vocbase, "_jobs");
+bool UpgradeTasks::setupJobs(
+    TRI_vocbase_t& vocbase,
+    arangodb::velocypack::Slice const& slice
+) {
+  return ::createSystemCollection(&vocbase, "_jobs");
 }
 
-bool UpgradeTasks::createJobsIndex(TRI_vocbase_t* vocbase, VPackSlice const&) {
-  ::createSystemCollection(vocbase, "_jobs");
-  ::createIndex(vocbase, "_jobs", Index::TRI_IDX_TYPE_SKIPLIST_INDEX,
-                {"queue", "status", "delayUntil"},
-                /*unique*/ true, /*sparse*/ true);
-  ::createIndex(vocbase, "_jobs", Index::TRI_IDX_TYPE_SKIPLIST_INDEX,
-                {"status", "queue", "delayUntil"},
-                /*unique*/ true, /*sparse*/ true);
+bool UpgradeTasks::createJobsIndex(
+    TRI_vocbase_t& vocbase,
+    arangodb::velocypack::Slice const& slice
+) {
+  ::createSystemCollection(&vocbase, "_jobs");
+  ::createIndex(
+    &vocbase,
+    "_jobs",
+    Index::TRI_IDX_TYPE_SKIPLIST_INDEX,
+    {"queue", "status", "delayUntil"},
+    /*unique*/ true,
+    /*sparse*/ true
+  );
+  ::createIndex(
+    &vocbase,
+    "_jobs",
+    Index::TRI_IDX_TYPE_SKIPLIST_INDEX,
+    {"status", "queue", "delayUntil"},
+    /*unique*/ true,
+    /*sparse*/ true
+  );
+
```

Comment:
Are you using clang-format for this ?

Reply:
"clang-format"? I didn't change the arguments, only changed the pointer to a reference.

Reply:
I know but you are reformatting a lot of stuff, for which we generally try to use the automated tool `clang-format`

Reply:
I format by hand for readability